### DISCUSSION TITLE
[Feature] Expanded Embedded Metadata Scanning

### DISCUSSION THREAD

**Discussion by grainsoflight** [immich-app/immich#6237] [Feature Request]
### The feature

Very excited to see how this project grows, it looks great. The only major downside for me at the moment is the very limited metadata support. I have thousands of family photos that have been tagged (as keyword) by lightroom with person names, as well as custom dates and descriptions. All of this is saved to the png file and is readable by other programs. It would be great if this more detailed metadata could be incorporated so that I could have family members search through files by the existing keywords.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by RedStop** [comment]
Especially to read the comment added by exiftool's -comment="This is some comment."
